src/Aspire.Hosting.Azure.AIFoundry/AIFoundryModel.Generated.cs

```diff
@@ -490,6 +615,28 @@ public static class OpenAI
 
     }
 
+    /// <summary>
+    /// Models published by Stability AI.
+    /// </summary>
+    public static class StabilityAi
```

Comment:
To match .NET naming standards, this should be StabilityAI. Should only pascal case when acryonums are 3 characters or more, e.g. System.IO vs System.Xml